godownloader.sh

```diff
@@ -1,399 +0,0 @@
-#!/bin/sh
```

Comment:
could you tell me the reason why this is removed? 🙂

Reply:
sure it is not used anywhere, or documented for use, but yeah I missed a spot as it is still in .editorconfig

Reply:
removed from .editorconfig also

Reply:
this is used in https://github.com/tetratelabs/getenvoy.io/blob/master/site/static/_redirects#L1 😄 😄

Reply:
I'll revert this.. generally, we shouldn't host generated shell scripts in repos they aren't used. Also this is a horrible name for the shell script.

Reply:
opened https://github.com/tetratelabs/getenvoy.io/issues/157